package.json

```diff
@@ -74,7 +74,7 @@
     "**/recursive-readdir/minimatch": "5.1.1",
     "fastify": "3.29.4",
     "loader-utils": "2.0.4",
-    "terser": "5.16.0",
```

Comment:
`terser` is in the resolutions list. Do we want to update it?

Reply:
If everything works, yes. It's in the resolutions list because the package that uses it couldn't be updated.